pkg/util/timeutil/now_unix.go

```diff
@@ -22,5 +22,5 @@ import (
 
 // Now returns the current local time.
```

Comment:
this comment still talks about local time.

Reply:
Done.